src/lib/accessibility/Behaviors/Attachment/attachmentBehavior.ts

```diff
@@ -0,0 +1,25 @@
+import { Accessibility } from '../../types'
+import * as keyboardKey from 'keyboard-key'
+
+/**
+ * @specification
+ * Adds attribute 'tabIndex=0' to 'root' component's part.
+ * Performs click action with 'Enter' and 'Spacebar' on 'root'.
+ */
+
+const attachmentBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      tabIndex: '0',
```

Comment:
wouldn't be more safer to set rather it for tabIndex=-1 
Now probably focusZone handle it in some way... otherwise tabIndex=0 means it will be in browser Tab order, always..